backend-worker/worker.py

```diff
@@ -427,26 +432,32 @@ def pixelpercents_under_parcel(parcel_wkt_epsg3857, source_raster_path):
     return_values = {}
     n_values_under_parcel = numpy.sum(counts)
     for lulc_code, pixel_count in zip(values_under_parcel, counts):
-        return_values[lulc_code] = round(
+        # cast lulc_codes to int from numpy_int16 for future json dump call
+        # which does not allow numpy types for keys
+        return_values[int(lulc_code)] = round(
```

Comment:
Oh that's interesting!  I would have thought numpy int16 would cast automatically to a python int.